### DISCUSSION TITLE
webhook condition ignore changes in custom field

### DISCUSSION THREAD

**Discussion by powerscooter** [netbox-community/netbox#14733] [General]
Hello everyone,

I have set up a webhook which sends data to a webhook server when creating or updating a console port.

To see in Netbox if the configuration was successful I have created a custom field which is currently called "SYNC".

If the console server has been configured correctly, a "YES" is written to the field, if there is an error, "Error" is written to the field.

Because I now customise the SYNC field after configuring it via API, I always have an "UPDATE" event. Which causes the webhook to be executed again and again.

Is there a way to ignore changes in a specific custom field?

Thanks!

<img width="764" alt="image" src="https://github.com/netbox-community/netbox/assets/18429672/b526e5bb-bde3-4fe4-862e-f9d909280ca1">

**Comment by markkuleinio** [comment]
I do it so that in the webhook handler I check the username of the action. If the username is the automation username itself, the webhook handler does nothing.

**Comment by powerscooter** [comment]
I have already considered this, but then Netbox sends unnecessary traffic to my webhook server.

Is this possible with the webhook conditions?
